packages/block-library/src/navigation/index.php

```diff
@@ -267,6 +267,10 @@ function render_block_core_navigation( $attributes, $content, $block ) {
 		$inner_blocks_html .= $inner_block->render();
 	}
 
+	if ( true === $is_list_open ) {
```

Comment:
```suggestion
	if ( $is_list_open ) {
```

Reply:
Yes. Makes sense. I guess the if statements above (https://github.com/WordPress/gutenberg/blob/834a20a4fbcb86c26a0252bff727aaa6314cc2ac/packages/block-library/src/navigation/index.php#L259-L266) should also be updated to be consistent. You want me to add it to this PR?

Reply:
I guess you could, if you don't mind. Since `$is_list_open` is assigned only `boolean` values and is initialized with `false`, we can loose the `=== true/false`.